pkg/auth/manager_test.go

```diff
@@ -175,6 +176,13 @@ type fakeIPCache struct {
 	nodeIdMappings map[uint16]string
 }
 
+func (r *fakeIPCache) DumpNodeIDs() []*models.NodeID {
```

Comment:
nit: Consider renaming this type to `fakeNodeIDHandler`

Reply:
@meyskens thanks a lot for dealing with that during my PTO!